### DISCUSSION TITLE
[question/docs] Relationship with React

### DISCUSSION THREAD

**Discussion by dandv** [alephjs/aleph.js#111] [General] [answered]
Hi, I've been using React for a few years but I'm new to Deno and I've just discovered this project. Looks very promising!

After reading through the website and this repo for a few minutes, I'm unclear as to what the relationship with React is. Is this a port of Rect for Deno? Or is it a React-like framework? What is the extent of compatibility with the React ecosystem components?

Would be great to clarify these things upfront. Thanks!

**Comment by shadowtime2000** [comment, marked as answer]
@dandv Aleph is a Deno full stack framework - we currently only support React but from the restructure of our project we will be moving to become something like Razzle but for Deno and more full stacky. Think of Aleph like NextJS, React is just the view library we use.
